de-lib/src/main/java/org/iplantc/de/diskResource/client/presenters/grid/GridViewPresenterImpl.java

```diff
@@ -568,9 +569,10 @@ public void setParentPresenter(DiskResourceView.Presenter parentPresenter) {
 
     @Override
     public void setSelectedDiskResourcesById(List<? extends HasId> diskResourcesToSelect) {
-        SelectDiskResourceByIdStoreAddHandler diskResourceByIdStoreAddHandler = new SelectDiskResourceByIdStoreAddHandler(diskResourcesToSelect,
-                                                                                                                          this);
-        HandlerRegistration diskResHandlerReg = listStore.addStoreAddHandler(diskResourceByIdStoreAddHandler);
+        SelectDiskResourceByIdStoreAddHandler diskResourceByIdStoreAddHandler =
```

Comment:
Minor thing - I thought there was a setting in IntelliJ to only format modified lines of code.  I'm having trouble finding it now, but I'll let you know if I find it again.